.github/workflows/micro-main.yml

```diff
@@ -0,0 +1,38 @@
+name: testing with Snowplow Micro
+
+on:
+
+  push:
+    branches:
+      - '*'
+
+jobs:
+
+  testing:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
```

Comment:
do we need to use the matrix strategy if we're not using multiple configuration options?

from the [docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) 
> You can specify a matrix by supplying an array for the configuration options. For example, if the runner supports Node.js versions 10, 12, and 14, you could specify an array of those versions in the matrix.

Reply:
Thanks, I added it at the time to be flexible, but you are right it makes more sense to only add what is needed. 

I'll change it 🙌🏽